test/include.cpp

```diff
@@ -8,6 +9,7 @@
 #include <ghc/filesystem.hpp>
 #include <rapidjson/document.h>
 #include <jni/jni.hpp>
+#include <supercluster.hpp>
```

Comment:
Supercluster depends on kdbush.hpp internally - thus why I'm not adding it explicitly here.

Reply:
I'd prefer if we err on the side of over-testing.